packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -154,12 +154,22 @@ export class RelationshipsController extends TransportController {
 
         const { requestCipher, requestContent } = await this.prepareRequest(secretId, template, parameters.content);
 
-        const backboneResponse = (
-            await this.client.createRelationship({
-                content: requestCipher.toBase64(),
-                relationshipTemplateId: template.id.toString()
-            })
-        ).value;
+        let backboneResponse: BackboneGetRelationshipsResponse;
+        try {
+            backboneResponse = (
+                await this.client.createRelationship({
+                    content: requestCipher.toBase64(),
+                    relationshipTemplateId: template.id.toString()
+                })
+            ).value;
+        } catch (error: any) {
+            if (error.code === "error.platform.validation.relationship.peerIsToBeDeleted") {
+                throw new Error(
+                    "The Identity who created the RelationshipTemplate is currently in the process of deleting itself. Thus, it is not possible to establish a Relationship to it."
+                );
+            }
+            throw error;
+        }
```

Comment:
you don't need try..catch - we have a result for this. If you don't call `.value` on the result of `createRelationship` you can just look if everything went through.